README.md

```diff
@@ -69,6 +69,43 @@ This assumes you make your build dir a subdirectory of the ceph.git
 checkout.  If you put it elsewhere, just replace .. above with a
 correct path to the checkout.
 
+To build only certain targets use:
+
+    $ make [target name]
+
+To install:
+
+    $ make install
+ 
+CMake Options
+-------------
+
+If you run the `cmake` command by hand, there are many options you can
```

Comment:
Any chance on an exhaustive list of -D options? Can we expect sys admins to know how to interpret CMakeLists.txt files to figure it all out?